examples/example_download_network_using_a_polygon.ipynb

```diff
@@ -0,0 +1,398 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " ### Download a Network using a polygon"
```

Comment:
The first header should just be a `#` .
Keep in mind that you need to adhere to the title hierarchy so that we can later on add this example into the documentation files (via sphinx)
```suggestion
    " # Download a Network using a polygon"
```